sim/midas/src/main/scala/midas/passes/fame/MultiThreadFAME5Models.scala

```diff
@@ -141,8 +141,9 @@ object MultiThreadFAME5Models extends Transform {
     // Populate keys from annotations, values from traversing statements
     val fame5RawInstances = new mutable.LinkedHashMap[OfModule, mutable.LinkedHashSet[Instance]]
     state.annotations.foreach {
-      case FirrtlEnableModelMultiThreadingAnnotation(ModuleTarget(_, m)) =>
-        fame5RawInstances(OfModule(m)) = new mutable.LinkedHashSet[Instance]
+      case FirrtlEnableModelMultiThreadingAnnotation(it) =>
+        // TODO: why not use instance name from here?
```

Comment:
I put this TODO in, but this just falls under the umbrella of cleaning up how we handle local instance targets / dedup / general annotation flow.